index.js

```diff
@@ -16,11 +16,58 @@ const nightbot = require('./custom-endpoints/nightbot');
 
 const schema = buildSchema(typeDefs);
 
+async function sha256(message) {
+    // encode as UTF-8
+    const msgBuffer = new TextEncoder().encode(message);
+
+    // hash the message
+    const hashBuffer = await crypto.subtle.digest('SHA-256', msgBuffer);
```

Comment:
Something is odd here, I get the following error locally and when deployed to Cloudflare:

```text
TypeError: Cannot read properties of undefined (reading 'digest') at line 0, col -2
```